libsrc/common/lz4.s

```diff
@@ -0,0 +1,283 @@
+;
+; Lauri Kasanen, 6 Jun 2017
+; (C) Mega Cat Studios
+; An optimized LZ4 decompressor
+;
+
+        .importzp       sp, sreg, regsave, regbank
+        .importzp       tmp1, tmp2, tmp3, tmp4, ptr1, ptr2, ptr3, ptr4
+        .macpack        longbranch
+        .import         memcpy_upwards,pushax,popax
+        .export         _decompress_lz4
+
+_out = regsave
+_written = regsave + 2
+_tmp = tmp1
+_token = tmp2
+_offset = ptr3
+_in = sreg
+_outlen = ptr4
```

Comment:
Is there a specific reason why you added an '_' prefix to all the symbols? An '_' prefix sort of implies the intend the make something visible from C - which isn't the case here.